docs/slides.md

```diff
@@ -25,18 +25,26 @@ Carl Pacey and Mark Tse
 
 # Goals
 
+Get some practice breaking up PRs:
+
+1. _After_ the code is written
+1. _Before_ the code is written
+
 ---
 
 # Techniques
 
-* Add a new class, method, property, etc
+1. Add a new class, method, property, etc
   * Test it, but don't use it
-* Add new tests to existing code
-* Refactor existing code
-  * Specifically to make your next change easier, safer, or smaller
-  * Try to limit or isolate test changes
+1. Add new tests to existing code
+  * Tr to limkit or isolate changes to production code
+1. Refactor existing production code
+  * Specifically to make your next change(s) easier, safer, or smaller
+  * Try to limit or isolate changes to test code
   * Try to avoid (or isolate) functional changes
-* Functional changes are smaller after you make the above non-functional changes first
+1. Functional changes
+  * Thesae get small after you've done the above
+  * New tests end up being for new production code

```

Comment:
Add "only" to the end? e.g. "New tests end up being for new production code only"